bindings/rust/src/lib.rs

```diff
@@ -283,15 +316,19 @@ impl KZGCommitment {
         hex::encode(self.bytes)
     }
 
-    pub fn blob_to_kzg_commitment(blob: Blob, kzg_settings: &KZGSettings) -> Self {
+    pub fn blob_to_kzg_commitment(blob: Blob, kzg_settings: &KZGSettings) -> Result<Self, Error> {
         let mut kzg_commitment: MaybeUninit<KZGCommitment> = MaybeUninit::uninit();
         unsafe {
-            blob_to_kzg_commitment(
+            let res = blob_to_kzg_commitment(
```

Comment:
This was a problem before too. It didn't check the return value. We wouldn't have known if it failed.

Reply:
Good catch. Let's add `#[must_use]` on top of `pub enum C_KZG_RET` in `bindings.rs` so that the compiler warns us if we are ignoring any `C_KZG_RET` in the future.

Reply:
Done in 9559dd081f9044013b164f08a33347da6a31b4f3.

Reply:
Thank you!